validator.js

```diff
@@ -52,6 +52,8 @@
     var isbn10Maybe = /^(?:[0-9]{9}X|[0-9]{10})$/
       , isbn13Maybe = /^(?:[0-9]{13})$/;
 
+    var macAddressMaybe = /^([0-9a-fA-F][0-9a-fA-F]:){5}([0-9a-fA-F][0-9a-fA-F])$/;
```

Comment:
Why `macAddressMaybe` and not `macAddress`? The IP and ISBN regexes are named as such because the functions contain additional validation.